astarte-device-sdk-derive/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Feat
+- Add macro to implement the `FromEvent` trait on a generic struct.
```

Comment:
We follow [keepachangelog](https://keepachangelog.com/en/1.0.0/#how), use `Added` for introducing a new features.
```suggestion
### Added
- Macro to implement the `FromEvent` trait on a generic struct.
```

Reply:
My bad, fixed